modules/contacts/client/components/RemoveContact.js

```diff
@@ -21,23 +21,23 @@ export default function RemoveContact({
 
   const isFromMe = contact.userFrom === selfId;
   const isConfirmed = contact.confirmed;
-  const created = new Date(contact.created);
+  const date = new Date(contact.created);
 
   if (isConfirmed) {
     // Remove confirmed contact
     labelTitle = t('Remove contact?');
     labelConfirm = t('Yes, remove contact');
-    labelTime = t('Connected since {{created, MMM D, YYYY}}', { created });
+    labelTime = t('Connected since {{date, LL}}', { date });
```

Comment:
I think this was broken already just because it was `created` and not `date` which causes the right filter to work in i18n-next.